### PR TITLE
Feat: Allow user to change the project for devtron apps

### PR DESCRIPTION
**Pull request by vikramdevtron**
# Description

Allow user to change the project for devtron apps.
Provide app update api where labels and project can be modified.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] App update - change project

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas
* [x] I have tested it for all user roles
* [ ] I have added all the required unit/api test cases

### CODE REVIEW

File: util/rbac/EnforcerUtil.go [hunk 1]
```diff
@@ -335,3 +336,15 @@ func (impl EnforcerUtilImpl) GetHelmObjectByProjectIdAndEnvId(teamId int, envId
 	}*/
 	return fmt.Sprintf("%s/%s/%s", strings.ToLower(team.Name), environmentIdentifier, "*")
 }
+
+func (impl EnforcerUtilImpl) GetAppRBACNameByTeamIdAndAppId(teamId int, appId int) string {
+	application, err := impl.appRepo.FindById(appId)
+	if err != nil {
+		return fmt.Sprintf("%s/%s", "", "")
```
Comment (kartik-579) [inline util/rbac/EnforcerUtil.go:343]: please add log

File: util/rbac/EnforcerUtil.go [hunk 2]
```diff
@@ -335,3 +336,15 @@ func (impl EnforcerUtilImpl) GetHelmObjectByProjectIdAndEnvId(teamId int, envId
...
+	}
+	team, err := impl.teamRepository.FindOne(teamId)
+	if err != nil {
+		return fmt.Sprintf("%s/%s", "", "")
```
Comment (kartik-579) [inline util/rbac/EnforcerUtil.go:347]: please add log

File: pkg/bean/app.go [hunk 3]
```diff
@@ -581,3 +581,9 @@ type AppMetaInfoDto struct {
 type AppLabelsJsonForDeployment struct {
 	Labels map[string]string `json:"appLabels"`
 }
+
+type UpdateProjectBulkAppsRequest struct {
+	AppId  []int `json:"appId"`
```
Comment (kartik-579) [inline pkg/bean/app.go:586]: can change to appIds

Comment (vikramdevtron) [reply pkg/bean/app.go:586]: done

**Review by kartik-579 — APPROVED** [review]

**Pull request merged**